mpisppy/utils/config.py

```diff
@@ -958,32 +958,12 @@ def tracking_args(self):
                             domain=int,
                             default=0)
 
-
     def wxbar_read_write_args(self):
```

Comment:
Is this idea that this method would eventually go away so that `generic_cylinders` or any other clients directly call the `add_options_to_config` on the module?

Reply:
Yes.